app/app.js

```diff
@@ -1,11 +1,15 @@
 const express = require('express');
-const { getAllTopics } = require('./app.controller');
+const { getAllTopics, getApiEndpoints } = require('./app.controller');
 const { handleServerErrors} = require('../errors');
 
 const app = express();
 
+app.use(express.json())
```

Comment:
We don't technically need this middleware function yet as we only have a get request. Please remove before you merge to main.

Reply:
alright hannah. thanks
will do so now
sending forth another ticket too 😃 for a pull request review